.github/actions/land-blocking/find-lbt-images.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Copyright (c) The Libra Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+# adapted from --build-all-cti option from libra/docker/build-aws.sh, which is how
+# land blocking test builds images before running cluster-test
+REPOS=(libra_validator libra_cluster_test libra_init libra_safety_rules)
+# the number of commits backwards we want to look
+END=5
```

Comment:
We probably want this to be at least ~50.
Reason is that imagine we disabled LBT for some time, and then re-enabled it - it might need to look a bit deep to find proper commit (note that some PRs have multiple commits, so single PR can easily add 5 commits)